traffic_ops/traffic_ops_golang/monitoring.go

```diff
@@ -1,90 +1,45 @@
-// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
sorry I got lost when I was popping back and forth for this review, I see what you mean now